src/learn/index.rst

```diff
@@ -13,3 +13,4 @@ Learn about Nextstrain and related background information.
    About Nextstrain <about-nextstrain>
    interpret/index
    pathogens/index
+   Nextclade analysis <nextclade>
```

Comment:
See above comment, this doesn't work as is and so should be converted to a link or else the whole algorithm section needs to get fetched and included in the right directory
```suggestion
   Nextclade analysis <https://docs.nextstrain.org/projects/nextclade/en/latest/user/algorithm/index.html>
```